### DISCUSSION TITLE
Is it wise to just test this on a Pixel 6?

### DISCUSSION THREAD

**Discussion by stemid** [openandroidinstaller-dev/openandroidinstaller#479] [Q&A]
Pixel 6 is not in the supported devices list so is it unwise to just try and test it? I have a spare phone.

**Comment by tsterbak** [comment]
Hey, thanks for your interest in the project :)

The installer will only work for a device, if a specific config file for this device is added in the installer. So it won't work for you even if you try.

The process of adding a new config file is not too hard and can be found here: https://github.com/openandroidinstaller-dev/openandroidinstaller/blob/main/docs/how_to_contribute_your_own_installation_configurations.md

But since the installer relies on TWRP which is not available for this device yet, I think it will be hard to add support right now.
